tests/testthat/test-profile_emissions.R

```diff
@@ -456,7 +386,7 @@ test_that("with some match preserves unmatched products (#193)", {
   expect_equal(unique(product$activity_uuid_product_uuid), c("unmatched", uuid))
 })
 
-test_that("with no match preserves unmatched products (#193)", {
+test_that("with no match errors gracefully", {
```

Comment:
FIXME? I may be introducing a regression to #193. I need to investigate more so be sure.